src/Route.elm

```diff
@@ -19,34 +21,35 @@ type Route
     | Options (Maybe String) (Maybe String) (Maybe String) (Maybe Int) (Maybe Int) (Maybe String)
 
 
+{-| Fixes issue with elm/url not properly escaping string
+-}
+queryString : String -> Query.Parser (Maybe String)
+queryString =
+    Query.map (Maybe.andThen Url.percentDecode) << Query.string
+
+
 parser : Url.Parser.Parser (Route -> msg) msg
 parser =
     Url.Parser.oneOf
-        [ Url.Parser.map
-            Home
-            Url.Parser.top
-        , Url.Parser.map
-            NotFound
-            (Url.Parser.s "not-found")
-        , Url.Parser.map
-            Packages
+        [ Url.Parser.map Home Url.Parser.top
+        , Url.Parser.map NotFound (Url.Parser.s "not-found")
+        , Url.Parser.map Packages
```

Comment:
I took a liberty and reformatted these parts a bit. This is all elm-format compatible.